/home/runner/work/_actions/ros-industrial/industrial_ci/master/.github/action.sh: No such file or directory

Hi, thank you for developing and maintaining this essential ROS quality assurance tool.

Apologies for my inexperience using GitHub Actions. This error is consistently encountered under the following scenario:
# Error :warning: 
```bash
/__w/_temp/c678a811-8165-41e2-a8c1-8a4d2dd6ce5f.sh: line 1: /home/runner/work/_actions/ros-industrial/industrial_ci/master/.github/action.sh: No such file or directory
Error: Process completed with exit code 127.
```


## **Scenario**: 

Using **custom public docker** for use in GitHub Actions workflow CI. :whale: 

```yaml
name: CI

on: [push, pull_request]
jobs:
  industrial_ci:
    strategy:
      matrix:
        env:
          - {ROS_DISTRO: foxy, ROS_REPO: testing}
          - {ROS_DISTRO: foxy, ROS_REPO: main}
    env:
      CCACHE_DIR: /github/home/.ccache

    runs-on: ubuntu-20.04
    # Use custom public docker image.
    container:
      image: cardboardcode/epd-foxy-base:emd_compatible
    steps:
      - uses: actions/checkout@v2
      - uses: actions/cache@v2
        with:
          path: ${{ env.CCACHE_DIR }}
          key: ccache-${{ matrix.env.ROS_DISTRO }}-${{ matrix.env.ROS_REPO }}
      # Run industrial_ci
      - uses: 'ros-industrial/industrial_ci@master'
        env: ${{ matrix.env }}

```

Not including the use of the public custom docker image in the above workflow `.yml` file **resolves the error**. 

```yaml
    container:
      image: cardboardcode/epd-foxy-base:emd_compatible
``` 

**However**, the point about using the docker image is to **address external dependencies** needed to build the ROS2 package. Therefore, the custom docker is needed.

Hoping to get some clarity on why this error keeps occuring when using custom docker.

--- a/action.yml
+++ b/action.yml
@@ -6,7 +6,7 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: ${{ github.action_path }}/.github/action.sh
+    - run: $GITHUB_ACTION_PATH/.github/action.sh
       shell: bash
       env:
         INPUT_CONFIG: ${{ inputs.config }}

